### DISCUSSION TITLE
I wasn't able to figure out which command changes the password from the web interface.

### DISCUSSION THREAD

**Discussion by Katranius** [wg-easy/wg-easy#231] [Ideas]
I had to remove wg-easy and reinstall.
Write the complete command, please.

**Comment by Katranius** [comment]
It would be nice to add a user for the web UI. View only. No removal.

**Comment by pheiduck** [comment]
> I had to remove wg-easy and reinstall. Write the complete command, please.

You can head over to the readme

> It would be nice to add a user for the web UI. View only. No removal.

So far, this feature has not been requested by many customers. If this changes, we will come back to it.
